Lib/threading.py

```diff
@@ -1570,7 +1570,7 @@ def _after_fork():
         # Dangling thread instances must still have their locks reset,
         # because someone may join() them.
         threads = set(_enumerate())
-        threads.update(_dangling)
+        threads.update(_dangling.copy())
```

Comment:
@vstinner worth adding a comment here akin to the asyncio one?

```suggestion
        # Looping over a WeakSet (_dangling) isn't safe as it can be updated from another
        # thread while we do so. Therefore we copy it prior to filtering.
        threads.update(_dangling.copy())
```

Reply:
Yes, please.